gui/launcher.cpp

```diff
@@ -255,6 +264,13 @@ void LauncherDialog::build() {
 	// I18N: Button caption. O is the shortcut, Ctrl+O, put it in parens for non-latin (~O~)
 	new ButtonWidget(this, _title + ".OptionsButton", _("Global ~O~ptions..."), _("Change global ScummVM options"), kOptionsCmd, 0, _c("Global ~O~pts...", "lowres"));
 
+	if (g_system->hasFeature(OSystem::kFeatureDLC)) {
+#if defined(USE_SCUMMVMDLC) && defined(USE_LIBCURL)
```

Comment:
What happens on Android? I think you need to show it when the feature is present, not when scummvmdlc is compiled